docs/05-low-code-spark/05-gems/02-transform/01-reformat.md

```diff
@@ -3,26 +3,29 @@ sidebar_position: 1
 title: Reformat
 ---
 
-Edit one or more column names or values, by using expressions and functions. It's useful when we need to extract only the required columns or make changes column-wise.
+Edits one or more column names or values, by using expressions and functions. It's useful when we need to extract only the required columns or make changes column-wise.
 
 
 ### Parameters
-| Parameter        | Meaning                                       | Required                                     |
-|:-----------------|:----------------------------------------------|:---------------------------------------------|
-| Dataframe        | Input dataframe on which changes are required | True                                         |
-| Target column    | Output column name                            | False                                        |
-| Expression       | Expression to compute target column           | False (Required if target column is present) |
+| Parameter     | Description                                   | Required                             |
+|:--------------|:----------------------------------------------|:-------------------------------------|
+| Dataframe     | Input dataframe on which changes are required | True                                 |
+| Target column | Output column name                            | False                                |
+| Expression    | Expression to compute target column           | Required if target column is present |
 
-:::info 
-To quickly select all the columns, click on the ADD ALL on the left side bar. 
-:::
 :::info
 If no columns are selected, then all columns are passed through to the output
 :::
+:::info 
+To quickly select all the columns, click on the ADD ALL on the left side bar.
```

Comment:
I don't think we need this part in the Reformat section, since the functionality is common across multiple gems